frontend/src/components/checkout/place-order/order-itmes/ItemsList.tsx

```diff
@@ -9,10 +9,9 @@ import { useShoppingCartHandlers } from "./handlers";
 import { product } from "../../../../contexts/shopping-cart-context/shoppingCartContextTypes";
 import { orderItemsListStyles } from "../../styles";
 
-const { cardContainer, itemImage, deleteButton } = orderItemsListStyles
+const { cardContainer, itemImage, deleteButton } = orderItemsListStyles;
```

Comment:
using Eslint could have warned you about this missing semicolon 😜 
maybe we can consider implementing it late 🤔

Reply:
I will do it later.

Reply:
I have a problem with prittier, it does not format on save, i blieve formating with prittier should add the semicolon.